samples/src/main/java/com/example/bigquery/RunLegacyQuery.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquery;
+
+// [START bigquery_query_legacy]
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.QueryJobConfiguration;
+
+public class RunLegacyQuery {
+
+  public static void runRunLegacyQuery() {
+    runLegacyQuery();
+  }
+
+  public static void runLegacyQuery() {
+    try {
+      // Initialize client that will be used to send requests. This client only needs to be created
+      // once, and can be reused for multiple requests.
+      BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
+
+      String query =
+          "SELECT corpus FROM [bigquery-public-data:samples.shakespeare] GROUP BY corpus;";
+
+      QueryJobConfiguration queryConfig =
+          // To use legacy SQL syntax, set useLegacySql to true.
+          QueryJobConfiguration.newBuilder(query).setUseLegacySql(true).build();
```

Comment:
Is this actually running the query? Or is it just configuring it? Shouldn't you need to do something to send it?

How do you process data received from the query?